pom.xml

```diff
@@ -198,6 +198,11 @@
       <artifactId>jersey-media-json-jackson</artifactId>
       <version>${jersey.version}</version>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>jvb-api-client</artifactId>
+      <version>2.1-227-g3ce5bf97</version>
```

Comment:
Do we want to create a jvb.version property?  Having the jvb-api-client and jitsi-videobridge versions out of sync seems like a recipe for weird errors (even though the latter is just for testing).

Reply:
Hmm, the flip side of that is that it makes it easier for someone to upgrade the jvb api client without realizing they're doing so.  I think that since the JVB is just used for testing (and has no interaction with the API code right now), it may feel better that they're versions are independent?

Reply:
The issue is that with how maven dependencies work, I think the jvb used for testing will be run with the version of jvb-api-common pulled in by jvb-api-client, not the version the jvb itself was built with.

Reply:
True, but I don't know that Jicof even starts the OSGi bundles when for the mock bridge, does it?

Reply:
No idea, but it sounds likely that it doesn't.

If you think it's okay, then ok.

Reply:
Ok, will keep an eye on it.